help/destinations/catalog/advertising/amazon-ads.md

```diff
@@ -77,10 +75,12 @@ To configure details for the destination, fill in the required and optional fiel
 *  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
 *  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
 *  **[!UICONTROL Amazon Ads Advertiser ID]**: Select the ID for the target Amazon Ads account used for the destination.
+      Note: After saving the destination configuration, you will not be able to change the Amazon Ads Advertiser ID, even if you re-authenticate through your Amazon account. To use a different Amazon Ads Advertiser ID, you must create a new destination connection.
```

Comment:
Please format the note as a note:

```
>[!NOTE]
>
>After saving the destination configuration, you will not be able to change the Amazon Ads Advertiser ID, even if you re-authenticate through your Amazon account. To use a different Amazon Ads Advertiser ID, you must create a new destination connection.
```

Reply:
Done.

Reply:
@vinvishy please use the exact formatting I provided in my comment. The `>` are mandatory for our formatting. You can copy and paste it from my comment.

Reply:
Done. Please take a look.